src/main/java/mate/academy/rickandmorty/service/CharacterService.java

```diff
@@ -0,0 +1,17 @@
+package mate.academy.rickandmorty.service;
+
+import java.util.List;
+import mate.academy.rickandmorty.model.CharacterEntity;
+import org.springframework.stereotype.Service;
+
+@Service
+public interface CharacterService {
+    List<CharacterEntity> getAll();
+
+    List<CharacterEntity> saveAll(List<CharacterEntity> characters);
+
+    CharacterEntity findById(Long id);
+
+    CharacterEntity findByExternalId(String externalId);
```

Comment:
actually, it's all methods that you need
```suggestion
    CharacterDto getRandomCharacter();

    List<CharacterDto> getCharactersByName(String name);
```